src/controllers/dlgprefcontroller.cpp

```diff
@@ -21,6 +21,17 @@
 #include "preferences/usersettings.h"
 #include "util/version.h"
 
+QString nameForPreset(const PresetInfo& preset);
+namespace {
+    bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
```

Comment:
Please don't indent within a namespace:
```c++
namespace {

bool presetInfoNameComparator(const PresetInfo &a, const PresetInfo &b) {
...
}

}  // namespace
```
Sorry this isn't mentioned in our style guide, but we do it this way. I'll add it!